simtools/nrn-patch.yaml

```diff
@@ -1,23 +1,7 @@
 - name: nrn-patch
 - website_url: ''
-- operating_system:
-  - windows: ''
-  - macOS: ''
-  - linux: ''
+- operating_system: ''
 - biological_level: ''
```

Comment:
This is essentially NEURON with some very small patches? Do we still include it as we have included Arbor and Abor-GUI, and link them as related/enhancements to the parent software?

Reply:
Not sure, on the border of relevance, I'd say…